nntrainer/layers/conv1d_layer.cpp

```diff
@@ -30,7 +30,6 @@ static constexpr size_t SINGLE_INOUT_IDX = 0;
 
 Conv1DLayer::Conv1DLayer(const std::array<unsigned int, 2> &padding_) :
   LayerImpl(),
-  padding(padding_),
```

Comment:
if it is not used, then we need to remove the `padding_ ` in param.

Reply:
Fixed